my-sms/spec/controllers/courses_controller_spec.rb

```diff
@@ -116,6 +116,26 @@
         is_expected.to be_success
         is_expected.to render_template('edit')
       end
+
+      it 'should propagate model errors to an error flash' do
+        expect { subject }.to change { flash[:error] }.from(nil).to(['End date must be after start date'])
+      end
+    end
+  end
+
+  describe 'DELETE #destroy' do
+    let!(:course) { create(:course) }
+    subject { delete :destroy, { id: course.to_param } }
+
+    # before { course }
+
+    it 'destroys the requested course' do
+      expect { subject }.to change(Course, :count).by(-1)
+    end
+
+    it 'redirects to the courses list' do
+      should be_redirect
+      should redirect_to(courses_url)
```

Comment:
nitpick - use expect matchers (again, my bad for setting a bad example earlier on)

```suggestion
      is_expected.to be_redirect
      is_expected.to redirect_to(courses_url)
```